core/spec/models/spree/product_spec.rb

```diff
@@ -244,9 +244,28 @@ def duplicate_extra(old_product)
 
         expect(product2.slug).to eq 'test-456'
       end
-
     end
 
+    context 'history' do
+      before(:each) do
+        @product = create(:product)
+      end
+
+      it 'should keep the history when the product is destroyed' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.